plugins/content_distribution/providers/tvinci/TvinciDistributionPlugin.php

```diff
@@ -2,7 +2,8 @@
 /**
  * @package plugins.tvinciDistribution
  */
-class TvinciDistributionPlugin extends KalturaPlugin implements IKalturaPermissions, IKalturaEnumerator, IKalturaPending, IKalturaObjectLoader, IKalturaContentDistributionProvider, IKalturaConfigurator
+class TvinciDistributionPlugin extends KalturaParentContributedPlugin implements IKalturaPermissions, IKalturaEnumerator, IKalturaPending, IKalturaObjectLoader,
```

Comment:
i think I got your point now  - some sort of a helper/util code that gives the children structure and can be called from the contribute of plugin itself 
if this is the case you are referring to - where would you place this code ? in the same location ?